audio/midiplayback.cpp

```diff
@@ -1020,5 +1020,5 @@ void _MIDI_fileclose(void)
 void _MIDI_readbuf(uint8_t* position, uint8_t* buf, uint16_t size)
 {
    fseek(midifile, (long long) position, 0);
-   fread(buf, 1, size, midifile);
+   size_t a = fread(buf, 1, size, midifile);
```

Comment:
`a` is unused?

Reply:
It would seem so

Reply:
there was a cpp flag, that I got from one of the comments here from some issue, that complained about `fread()` unused return value preventing compilation, dont remember which one tho, but being harmless I thought it was a good idea to let it be there.